### DISCUSSION TITLE
Contract Modification Notice - Make changes after previously submitting contract changes

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#817] [Q&A] [answered]
If a modification is made to the information of a contract modification already sent and published, what should happen? Send the modification again as if it were a new modification?

And if something is modified in the notice (not in the contract modifications section) and a contract modification has already been sent previously, should we generate an eform of type change?

Thanks in advance,
Alex

**Comment by YvesJo** [comment, marked as answer]
Hi,
Indeed, a Contract Modification on an already modified contract, should apply on the previous Contract Modification.
When it comes to  a change instead, the change should also apply to the latest notice containing the contract information (Result or Contract Modification.
KR

**Comment by AlexAS81** [comment]
The answer is not entirely clear to us.

We have found a case in which changes have been made to a contract modification and a change of the contract modification has been generated.

[https://ted.europa.eu/en/notice/-/detail/76564-2024](url)

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/45131d0c-e606-4e42-ac93-a0c14b28ea98)

This would clarify our first question.

We are left with the question of what type of notice we should generate if, after having sent a contract modification, a modification is made in another section that is not a contract modification.

We have not found examples of that type in production.

Maybe because there is no need to generate any type of new notice (change notice)?

Thanks in advance

**Reply by YvesJo** [reply]
If the update after the Contract Modification is not about modifying the Contract but fixing clerical mistakes, then a change should be used. If the update is about a Contract Modification, then a Contract Modification notice should be used.

**Reply by AlexAS81** [reply]
Okay, thank you very much.
